src/index.ts

```diff
@@ -1095,13 +1101,13 @@ export class ValidationService {
         this.logger.log("Watching for mutations");
     }
 
-    private observed(mutation: MutationRecord) {
+    private observed(mutation: MutationRecord, addNoValidate: boolean) {
         if (mutation.type === 'childList') {
             for (let i = 0; i < mutation.addedNodes.length; i++) {
                 let node = mutation.addedNodes[i];
                 this.logger.log('Added node', node);
                 if (node instanceof HTMLElement) {
-                    this.scan(node);
+                    this.scan(node, addNoValidate);
```

Comment:
If you make the other suggested change, this can probably be:

```suggestion
                    this.scan(node);
                    if (this.options.addNoValidate || true) {
                      for (var form of this.formInputs) {
                        form.setAttribute('novalidate', 'novalidate');
                      }
                    }
```

Reply:
`this.options.addNoValidate || true` will always be `true`! You mean `this.options.addNoValidate ?? true`.